google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
If you keep the extra FLEX_COMPAT then this is less of an issue and more academic (e.g there is no compat environment really).

Previously we did not have this, so any time we discussed the logic around flex compat we would stumble when trying to discuss what flex compat projects were in this enum (STD or FLEX). This had non-trivial consequences in the logic and could lead to bugs if not interpreted correctly.

In any case, I'd suggest keeping the FLEX_COMPAT for now and if Patrick is ok with the slight mischaracterization in the naming I can live with it for now.